src/parser/file-parser.js

```diff
@@ -76,7 +76,8 @@ function parseChecklyDirectory() {
     )
     return parseChecklySettings(parsedFiles)
   } catch (err) {
-    throw new Error(err.message)
+    console.error(err)
```

Comment:
Can we use `consola.error` instead?

Reply:
Yeah I was thinking about that actually, but consola wasn't imported here yet and I didnt want to do it necessarily just for this. But you're right - its much nicer and probably worth it :+1: